lib/calendarium-romanum/cli.rb

```diff
@@ -11,11 +11,23 @@ class CLI < Thor
     def query(date_str=nil)
       I18n.locale = options[:locale]
 
-      data_file = Data[options[:calendar]]
-      if data_file.nil?
-        die! 'Invalid calendar. See subcommand `calendars` for valid options.'
+      # This block is for specifying a full filesystem path
```

Comment:
Please remove all comments. The code is short and very straightforward, comments are not necessary.

Reply:
Done